bin/game/compat_1.2.nut

```diff
@@ -23,3 +23,9 @@ GSNews.Create <- function(type, text, company)
 {
     return GSNews._Create(type, text, company, GSNews.NR_NONE, 0);
 }
+
+GSBridge._GetName <- GSBridge.GetName;
```

Comment:
one last thing, can you add a comment above all of these (as per GSNews.Create) ?